droidlet/perception/semantic_parsing/nsp_transformer_model/caip_dataset.py

```diff
@@ -120,6 +120,12 @@ def __getitem__(self, idx):
             p_text, p_tree = t
         except ValueError as e:
             print(e)
+        try:
+            t = self.data[dtype][idx % len(self.data[dtype])]
+            p_text, p_tree = t
+        except ValueError as e:
+            print(e)
+        print(f"ptext: {p_text}, ptree: {p_tree}")
```

Comment:
should remove this

Reply:
whoops you reviewed this before I removed these files. Could you review again ?